dev/PushNotifications/PushNotificationTelemetry.h

```diff
@@ -31,9 +31,10 @@ class PushNotificationTelemetry : public wil::TraceLoggingProvider
                 TelemetryPrivacyDataTag(PDT_ProductAndServicePerformance),
                 _GENERIC_PARTB_FIELDS_ENABLED,
                 TraceLoggingHexUInt32(hr, "OperationResult"),
-                TraceLoggingBool(isAppPackaged, "IsAppPackaged"),
                 TraceLoggingWideString(to_hstring(remoteId).data(), "RemoteId"),
-                TraceLoggingWideString(GetAppUserModelId(), "AppUserModelId"));
+                TraceLoggingBool(usingLegacyImpl, "UsingLegacyImpl"),
+                TraceLoggingBool(IsPackagedApp(), "IsAppPackaged"),
```

Comment:
Isn't 'App' redundant here?

SUGGESTION: IsPackaged

Reply:
I kept the IsAppPackaged, to match the corresponding GetAppName.